pages/common/passwd.md

```diff
@@ -8,11 +8,7 @@
 
 - Change the password of the current user:
```

Comment:
If this looks good I'll commit it
```suggestion
- Change the password of any user:
```

Reply:
```suggestion
- Change the password of a specific user:
```

Reply:
We generally prefer simplicity, so in this case I'd right `...of a user:`.  When using simply `a` could be unclear, we add `specific` 👍🏻

Reply:
I think I prefer @navarroaxel's option, because it differentiates it from the previous example about changing the password of the current user.

Reply:
@navarroaxel I think we can apply it ourselves by now.